week-3/E-array-map/exercise.js

```diff
@@ -2,4 +2,6 @@
 // Write multiple solutions using different syntax (as shown in the README)
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
+var numbersByHundred = numbers.map(number => number * 100);
```

Comment:
This is correct, but the exercise asks you to try writing the solution multiple times using different syntax. Can you try that?